From 79a40cc4c3b5d6f8a90d9dc0ac5d2c40983764f7 Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@paulbarker.me.uk>
Date: Sun, 16 Jun 2013 23:27:27 +0100
Subject: [PATCH] Fix libopkg header installation.

When opkg is built outside the source directory, libopkg headers aren't all
installed as the search path for headers only includes the directory that make
is being ran in, not the source directory. So we fix this by adding
$(srcdir)/*.h to the list of header files to install.

Signed-off-by: Paul Barker <paul@paulbarker.me.uk>
Upstream-Status: Submitted
---
 libopkg/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libopkg/Makefile.am b/libopkg/Makefile.am
index 043c5c4..b6f4f7b 100644
--- a/libopkg/Makefile.am
+++ b/libopkg/Makefile.am
@@ -2,7 +2,7 @@
 AM_CFLAGS=-Wall -DHOST_CPU_STR=\"@host_cpu@\" -DBUILD_CPU=@build_cpu@ -DLIBDIR=\"@libdir@\" -DOPKGLIBDIR=\"@opkglibdir@\" -DOPKGETCDIR=\"@opkgetcdir@\" -DOPKGLOCKFILE=\"@opkglockfile@\" -DDATADIR=\"@datadir@\" -I$(top_srcdir) $(BIGENDIAN_CFLAGS) $(CURL_CFLAGS) $(GPGME_CFLAGS) $(PATHFINDER_CFLAGS)
 
 libopkg_includedir=$(includedir)/libopkg
-libopkg_include_HEADERS= *.h
+libopkg_include_HEADERS= *.h $(srcdir)/*.h
 
 
 opkg_libcore_sources = \
-- 
1.8.3

